src/lib/y2storage/autoinst_profile/drive_section.rb

```diff
@@ -117,6 +117,7 @@ def init_from_hashes(hash)
         super
         @type ||= default_type_for(hash)
         @use = use_value_from_string(hash["use"]) if hash["use"]
+        @enable_snapshots = true unless hash.key?("enable_snapshots")
```

Comment:
The documentation of this method says "_It only enforces default values for #type (:CT_DISK) and #use ("all") since the {AutoinstProposal} algorithm relies on them_". And indeed, IIRC the philosophy of these `init_from_hashes` methods of the `XXXSection` classes was not to set the default values in general. The `DriveSection` class is supposed to be a quite dumb class translating the hash-based format to something more useful, but turning nils into default values should not be part of this. Isn't it?

Reply:
Ok, fine with me. I will change it.